setup.py

```diff
@@ -141,7 +141,7 @@ def run(self):
 ]
 
 if (sys.version_info) >= (3, 6):
-    tests_requirements.append("black==19.3b0")
+    tests_requirements.append("black==19.10b0")
```

Comment:
black was not _working_ with Python 3.7.5, so I updated it to the latest version

Reply:
@mroutis Need to update https://github.com/iterative/dvc/blob/master/.restyled.yaml too

Reply:
@efiop , looks like restyled doesn't support black `19.10`:

> stderrUnable to find image 'restyled/restyler-black:v19.10b0' locally

Reply:
https://github.com/restyled-io/restylers/pull/50

It's going to take a while... I guess I'll create an issue and handle this in another PR. Meanwhile, I'll work with Python 3.6

Reply:
works fine for me on 3.7.3